content/en/real_user_monitoring/retention_analysis/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Retention Analysis
+kind: documentation
+private: true
+is_beta: true
+---
+
+<div class="alert alert-info">Retention Analysis is in private beta. If you have any questions, contact your customer success manager.</a></div>
+
+## Overview
+RUM Retention Analysis allows you to measure how often users are successfully returning to a page or action. By tracking user retention over time, you can gain insights around overall user satisfaction.
+
+A cohort is a group of users who participate in an initial event, such as clicking a link. A user in the cohort is considered retained if they subsequently complete a return event, such as clicking the same link again or clicking a **Proceed to Payment** button.
+
+The retention graph displays the percentage of users who completed the return event each week.
+
+{{< img src="real_user_monitoring/retention_analysis/example-retention-analysis-graph.png" alt="Example Retention Analysis graph" style="width:100%;" >}}
+
+## Build a graph
+
+To build a retention graph, navigate to **[UX Monitoring > Product Analytics > Retention Analysis][1]**, then follow the steps below.
+
+### 1. Define the initial event
+1. Select the view or action to act as the initial event for defining a group of users.
+2. Optionally, add filters such as the device used or the country of origin.
+    - If your initial event is a view, you can add any [view facets][2] or context facets.
+    - If your initial event is an action, you can add any [action facets][3] or context facets.
+
+### 2. Optionally, define the return event
+The return event defaults to a repeat of the original event. To use a different return event: 
+
+1. Change **Repeated the original event** to **Experienced a different event**.
+2. Select the view or action to act as the return event.
+3. Optionally, add any desired filter criteria.
```

Comment:
i wonder if we should specify an example for filter criteria. maybe saying "like geography"?